lib/matplotlib/tests/test_ticker.py

```diff
@@ -3,6 +3,7 @@
 import numpy as np
 from numpy.testing import assert_almost_equal, assert_array_equal
 import pytest
+import re
```

Comment:
This import belongs with `warnings`.

Reply:
I do not understand what you mean by *belongs with `warnings`*. Doesn't mean I do not have to import `re` ?

Reply:
According to [PEP8](https://www.python.org/dev/peps/pep-0008/#imports), imports are grouped by their origin: standard library, then third party, then local imports, separated by a blank line. `re` and `warnings` are standard library and should be grouped together.

Reply:
Thanks for explanation.

Changed. (I close this thread).